docker-compose.yml

```diff
@@ -1,15 +1,15 @@
 version: '2'
 services:
   cloudant:
-    image: ibmcom/cloudant-developer:latest
+    image: ibmcom/couchdb3:latest
     container_name: cloudant-testdb
```

Comment:
so even we use couchdb's image, we still name the container `cloudant-testdb`?

Reply:
@agnes512 yeah because we have another connector in https://github.com/strongloop/loopback-connector-couchdb2, we need to distinguish them.